### DISCUSSION TITLE
❤️  GitHub Models is now available in public preview

### DISCUSSION THREAD

**Discussion by Akash1134** [community/community#142244] [Other Feature Feedback, Questions, & Ideas]
## GitHub ❤️ AI engineers

If you’ve ever dreamed of leveraging cutting-edge AI models right from your GitHub workspace, your moment has arrived! GitHub Models has officially entered public preview! Dive into top AI models with a playground, API, and more—all designed to enhance your development experience.

<img width="730" alt="Screenshot 2024-10-29 at 17 13 29" src="https://github.com/user-attachments/assets/4e18bdf6-20a2-42e5-af69-3e58b1ce60e5">

Since the [announcement of GitHub Models almost three months ago](https://github.blog/news-insights/product-news/introducing-github-models/), we've shipped a number of enhancements and new models. [See the details here.](https://github.blog/changelog/2024-10-29-github-models-is-now-available-in-public-preview/)

You know what! **There’s no more waitlist, so try [GitHub Models](https://github.com/marketplace/models) out right away.**
 

### How to Get Started?

1. Play around in the[ GitHub Models Playground](https://github.com/marketplace/models).
2. Explore the Docs to learn about all the features available to you.

We want to hear from you! 💬 What experiments are you running? Any questions or feedback? Let’s chat below and shape the future of GitHub Models together! 🚀

**Comment by neverlous** [comment]
It is cool, I am trying

**Comment by hmorneau** [comment]
The o1 model always timeout in high effort mode.

**Comment by X2c2** [comment]
Hello everyone, thank you for this i am new. i wish everyone blessing.
